.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'GradleUp/shadow'
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: 21
+      - uses: gradle/actions/setup-gradle@v4
+        with:
+          cache-read-only: true
+      # Disable CC due to https://github.com/gradle/gradle/issues/22779
+      - run: ./gradlew release --no-configuration-cache
```

Comment:
This `release` task is for publishing all things, and but I can't ensure `gitPublishPush` works well for website deployment, can someone give me a hand to test and fix it?


https://github.com/GradleUp/shadow/blob/b3ce1927c7cc1d1c5bc0847d086e39d749b96108/gradle/publish.gradle#L31-L33

Reply:
15780f5c5f2f295640f8724261d94b3844c56106